changelogs/fragments/1073-action_plugin_does_not_clean_up_remote_temporary_files_after_completion.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - zos_unarchive - When unarchive uss file the module left temporary files.
```

Comment:
Would change this to "When unarchiving USS files the module left temporary files on the remote. Change now removes temporary files."

Reply:
I looked this over and the change was made, since Fernando is on holiday tomorrow, I think its fair to resolve this one on his behalf.